### PR TITLE
Post PR183 review fixes.

### PR DESCRIPTION
**Pull request by aosewski**
This PR addresses #183 review comments which didn't get in before merge.

* Rename `conv_fwd_util.hpp|cpp` file to `conv_util.hpp|cpp`
* Update `ConvParams` members to have `_` suffix.
* Update references to above

### CODE REVIEW

File: test/convnd_fwd/conv2d_fwd.cpp [hunk 1]
```diff
@@ -6,7 +6,7 @@
 
 #include "data_type.hpp"
 #include "element_wise_operation.hpp"
-#include "conv_fwd_util.hpp"
+#include "library/include/ck/library/utility/conv_util.hpp"
```
Comment (asroy) [inline test/convnd_fwd/conv2d_fwd.cpp:9]: #include "ck/library/utility/conv_util.hpp" should do

File: test/convnd_bwd_data/convnd_bwd_data.cpp [hunk 2]
```diff
@@ -13,9 +13,9 @@ int main()
     bool pass = true;
     // check 1d
     std::vector<ck::utils::conv::ConvParams> params;
-    params.push_back({1, 128, 128, 256, {1}, {14}, {2}, {1}, {0}, {0}});
-    params.push_back({1, 128, 128, 256, {3}, {28}, {1}, {1}, {1}, {1}});
-    params.push_back({1, 128, 128, 256, {1}, {3}, {1}, {1}, {0}, {0}});
+    params.push_back_({1, 128, 128, 256, {1}, {14}, {2}, {1}, {0}, {0}});
```
Comment (asroy) [inline test/convnd_bwd_data/convnd_bwd_data.cpp:16]: ```push_back```?

**Review by myamlak — DISMISSED** [review]
LGTM

File: test/convnd_bwd_data/convnd_bwd_data.cpp (hunk 2 above)
Comment (aosewski) [reply test/convnd_bwd_data/convnd_bwd_data.cpp]: Ohh I have missed this one. Thanks!

**Review by asroy — DISMISSED** [review]

**Review by asroy — APPROVED** [review]

**Pull request merged**